processes/dags/pubmed_migration.py

```diff
@@ -170,6 +170,18 @@ def pubmed_get_authors(pubmed_xml: dict) -> list:
         result = [get_author(a) for a in author_list]
     return result
 
+def pubmed_get_subjects(pubmed_xml: dict) -> list:
```

Comment:
Why are there 2 "pubmed_get_subjects"?
Are we sure the first 1 is always a publisher type? Is it possible that the first 1 may not be a publisher type?

Reply:
it's because there I added it to both local and AWS py files.

Reply:
It's either the first one, or the list is empty. for this data set, i think it is true.